src/components/Amount.tsx

```diff
@@ -32,7 +32,7 @@ export default function Amount({
   }
 
   return (
```

Comment:
collat ratio number should be vertically centered (left current code, right side design)

<img width="2438" alt="image" src="https://github.com/user-attachments/assets/291f2c8a-806f-491d-ac16-1614f278ab58">